rstudio_editor_theme_setup_tutorial.Rmd

```diff
@@ -0,0 +1,342 @@
+# RStudio Editor Theme Customization Tutorial
+
+Xiaoyang Song
+
+<style type="text/css">
+  body{
+  margin: auto;
+  font-size: 12pt;
+  font-family: 'Avenir', sans-serif;
+  }
+</style>
+<style>
+.center {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+}
+
+</style>
+
+```{r include=FALSE}
```

Comment:
Update to {r, include=FALSE}

Reply:
Hi, thanks for pointing that out. I just updated it.